tools/macaroon/cmd/brave-leo/premium_prod_time_limited_v2.yaml

```diff
@@ -0,0 +1,24 @@
+tokens:
+  - id: "brave-leo-premium"
+    version: 1
+    location: "leo.brave.com"
```

Comment:
What's the function of `location` - does this need to map to a domain that we're actually using? Right now aichat servers are hosted at aichat.bsg.brave.com, but I'm not sure if that's even relevant.